README.md

```diff
@@ -53,6 +53,16 @@ python setup.py install
     Runs the acutal migration. Will go through, the database table by table
     and migrate all rows to the target database.
 
+* `purge-tables`
+
+    Purges all tables in the target database. This subcommand can be executed
```

Comment:
So this seems like a lie, since we skip some tables. Do we want to say that here?